### PR TITLE
Fix migrations

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: migrations/20161112173153_add-eligibility.js
Comment (andrewrlee) [inline migrations/20161112173153_add-eligibility.js:4]: do you need to add these constraints after all the migrations have run? 

Is there a prod system for this that this could mess with? 

Another approach would be to ditch all the existing migrations, export the existing schema from prod as migration 0 and start again. Would need to reset migrations table etc..

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: migrations/20161112173153_add-eligibility.js
Comment (psoleckimoj) [reply migrations/20161112173153_add-eligibility.js:4]: I was wondering this as there are numerous tables that have never been added to the migrations hence me working backwards from them. Just a question of converting them to knex.

Comment (psoleckimoj) [reply migrations/20161112173153_add-eligibility.js:4]: @andrewrlee sorry had to rebase before it'd let me merge so needs another approval 😄

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**
